cmd/jwt/main.go

```diff
@@ -146,17 +146,17 @@ func verifyToken() error {
 		return data, nil
 	})
 
+	// Print an error if we can't parse for some reason
```

Comment:
I think we should check the error first before trying other stuff so I moved this up.